datasets/za/wanted/crawler.py

```diff
@@ -0,0 +1,60 @@
+from zavod import Context
+from lxml import html
+from urllib.parse import urlparse, parse_qs
+from typing import Dict
+import re
+
+from zavod import helpers as h
+
+REGEX_PATTERN = re.compile("(.+)\((.+)\)(.+)")
+
+
+def parse_cell(cell: html.HtmlElement):
+    result = {}
+    text = cell.text_content()
+    match = REGEX_PATTERN.match(text.strip())
+    if match:
+        name, crime, status = match.groups()
+        last_name, first_name = name.split(",", maxsplit=1)
+        result["first_name"] = first_name.strip()
+        result["last_name"] = last_name.strip()
+        result["crime"] = crime.strip()
+        result["status"] = status.strip()
+        details_id = cell.xpath(".//a/@href")[0]
+        source_url = (
+            f'https://www.saps.gov.za/crimestop/wanted/{details_id}'
+        )
```

Comment:
for future reference, here's another way to do this

https://github.com/opensanctions/opensanctions/blob/main/datasets/co/funcion_publica/crawler.py#L199

Reply:
Thanks!

Didn't knew about `make_links_absolute`. Always good to keep things DRY. 

Done in https://github.com/opensanctions/opensanctions/pull/533/commits/29b2f955479072c393cf0ddecff19ec031353a8b